### PR TITLE
Complete create customer fields

### PR DESCRIPTION
**Pull request by Cyrelc**

### CODE REVIEW

File: resources/views/drivers/drivers.blade.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+@endsection
+
+@section ('script')
+
+<script type='text/javascript'>
+
+	function childRow(details) {
+		var data = JSON.parse(details);
+
+		var thisBill = 'editDriver' + data.number;
```
Comment (MrMagaw) [inline resources/views/drivers/drivers.blade.php]: var thisBill? Should be var thisDriver.

File: resources/views/invoices/invoices.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+@endsection
+
+@section ('script')
+
+<script type='text/javascript'>
+
+	function childRow(details) {
+		var data = JSON.parse(details);
+
+		var thisCust = 'editInvoice' + data.number;
```
Comment (MrMagaw) [inline resources/views/invoices/invoices.blade.php]: var thisCust? Should be var thisInvoice.

**Comment by MrMagaw** [comment]
So close!  
Also, I'm sorry I haven't gotten around to the database for these sections yet.

**Comment by Cyrelc** [comment]
No worries, I'll fix that in a bit, and actually if you do get time I'd
much rather have the controllers completed

**Pull request merged**